dftimewolf/lib/processors/turbinia_gcp.py

```diff
@@ -18,80 +18,41 @@
 
 # pylint: disable=no-member
 
-
-class TurbiniaProcessor(module.BaseModule):
-  """Processes Google Cloud (GCP) disks with Turbinia.
+# pylint: disable=abstract-method
+class TurbiniaProcessorBase(module.BaseModule):
+  """Base class for processing with Turbinia.
 
   Attributes:
+    turbinia_config_file (str): Full path to the Turbinia config file to use.
     client (TurbiniaClient): Turbinia client.
-    disk_name (str): name of the disk to process.
     instance (str): name of the Turbinia instance
-    project (str): name of the GPC project containing the disk to process.
+    project (str): name of the GCP project containing the disk to process.
+    run_all_jobs (bool): Whether to run all jobs instead of a faster subset.
```

Comment:
What jobs does this include? It would be nice to be able to pass through a job whitelist to Turbinia so one can choose (as an advanced feature). 

In any case, let's be clear what jobs are included/excluded here.

Reply:
We only exclude slow-running jobs here ('StringsJob', 'BinaryExtractorJob', 'BulkExtractorJob', 'PhotorecJob'), added to the docstring. I guess I'll never have the full list of jobs that are actually run because this is dependent on the Turbinia deployment.

I think it would be better to rename this parameter into `skip_slow_jobs`. I opened https://github.com/log2timeline/dftimewolf/issues/351 to track this.

Reply:
👍